bridge/client/app/_components/_dialogs/ktb-deletion-dialog/ktb-deletion-dialog.component.ts

```diff
@@ -40,12 +51,17 @@ export class KtbDeletionDialogComponent implements OnInit, OnDestroy {
     });
   }
 
+  ngAfterViewInit(): void {
+    this.formInput?.nativeElement.focus();
+    this._changeDetectorRef.detectChanges();
```

Comment:
Is the change detector needed here? This component does not have the OnPush logic as far as I can see

Reply:
Without the `detectChanges`, I was getting [`NG0100: Expression has changed after it was checked`](https://angular.io/errors/NG0100).